extensions/amp-google-assistant-assistjs/0.1/test/validator-amp-google-assistant-assistjs.html

```diff
@@ -0,0 +1,20 @@
+<!doctype html>
```

Comment:
Please insert a copyright statement and short description header comment for this tests file. Take a look at some of the other extensions for an example.

Reply:
Done